src/services/EnvironmentVariables.ts

```diff
@@ -229,6 +233,19 @@ export class EnvironmentVariables {
 					this.RESOURCES_TTL_SECS = 1209600;
 					logger.warn("RESOURCES_TTL_SECS env var is not set or below 12 days. Setting to minimum - 12 days.");
 				}
+				if (!this.FETCH_YOTI_SESSION_BACKOFF_PERIOD_MS
+					|| this.FETCH_YOTI_SESSION_BACKOFF_PERIOD_MS.trim().length === 0
+					|| +this.FETCH_YOTI_SESSION_BACKOFF_PERIOD_MS.trim() === 0
+					|| +this.FETCH_YOTI_SESSION_BACKOFF_PERIOD_MS.trim() >= 60000) {
```

Comment:
Sorry why do we care if it's over 60000?

Reply:
no specific reason, but good to have a check on max value. Can remove/change it if unnecessary.